crates/common/rlp/src/decode.rs

```diff
@@ -145,6 +145,8 @@ impl Header {
         };
 
         if buf.remaining() < h.payload_length {
+            // maybe this check belongs in the derive impl for types that implement Decodable
+            println!("buf remaining: {}, payload length: {}", buf.remaining(), h.payload_length);
```

Comment:
Remove?

Reply:
Yep, just for visible CI logs

Reply:
ref https://github.com/paradigmxyz/reth/pull/1113#discussion_r1093820248